index.html

```diff
@@ -0,0 +1,107 @@
+<!-- Userform Title line  -->
+  <title>User Form</title>
+  <link rel="stylesheet" href="node_modules/bulma/css/bulma.css">
+  <link rel="stylesheet" href="css/mystyles.css">
+
+  <script defer src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
+  <script src="javascript/script.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<!-- Reference to user name and welcome page -->
+  <section class="section">
+    <div class="container">
+      <h1 class="is-size-3"><i class="far fa-user-circle is-size-1"></i> Welcome Back Peppa!</h1>
+    </div>
+  </section>
+
+<!-- Preferences title line  -->
+  <div class="container">
+    <h1 class="is-size-3 has-text-weight-bold">Preferences</h1>
+  </div>
+
+<!-- Month input line -->
+  <section class="section">
+    <div class="container">
+      <div class="field is-horizontal">
+        <div class="field-label is-large">
+          <label class="label">Month</label>
+        </div>
+        <div class="field-body">
+          <div class="field is-narrow">
+            <div class="control">
+              <div class="select is-rounded is-medium">
+<!-- Different drop down options for Month -->
+                <select>
+                  <option>Select Month</option>
+                  <option>January</option>
+                  <option>Febuary</option>
+                  <option>March</option>
+                  <option>April</option>
+                  <option>May</option>
+                  <option>June</option>
+                  <option>July</option>
+                  <option>August</option>
+                  <option>September</option>
+                  <option>October</option>
+                  <option>November</option>
+                  <option>December</option>
+                </select>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+<!-- Monthly budget input line -->
+      <div class="field is-horizontal py-3">
+        <div class="field-label is-large">
+          <label class="label">Monthly Budget</label>
+        </div>
+        <div class="field-body">
+          <div class="field has-addons">
+            <p class="control">
+              <a class="button is-rounded is-static">
+                CAD $
+              </a>
+            </p>
+            <p class="control">
+              <input class="input is-rounded" type="text">
+            </p>
+          </div>
+        </div>
+      </div>
+
+<!-- Budget category input line  -->
+      <div class="field is-horizontal">
+        <div class="field-label is-large">
+          <label class="label">Category</label>
+        </div>
+        <div class="field-body">
+          <div class="field is-narrow">
+            <div class="control">
+              <div class="select is-rounded is-medium">
+<!-- different dropdown options for category input  -->
+                <select>
+                  <option>Select Category</option>
```

Comment:
for future, im not too sure if we need to have a category dropdown. i know our tool will be having manual category inputs so we might need to add an amount under the selected category
code is great! 👍  this comment is more for future reference when we implement this page onto our dashboard